lib/modules/styleTweaks.js

```diff
@@ -164,6 +164,13 @@ module.options = {
 		}],
 		bodyClass: true,
 	},
+	hidesDomainLink: {
+		title: 'Hides Domain Link',
```

Comment:
Please move this title and description to `en.json`, same as all the other options here. https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/master/locales/locales/README.md#translating-modules